src/init.sh

```diff
@@ -46,6 +47,14 @@ function init_kw()
         find "$PWD/arch" -maxdepth 1 -mindepth 1 -type d -printf '%f\n'
       fi
     fi
+
+    if [[ -n "${options_values['remote']}" ]]; then
+      populate_remote_info "${options_values['remote']}"
```

Comment:
`populate_remote_info` might fail if the user gives a bad parameter. Check this function return and let the user know if they pass something wrong.